nncf/quantization/algorithms/post_training/algorithm.py

```diff
@@ -191,6 +191,8 @@ def _apply(self,
             # TODO (l-bat): Remove after OpenVINO Native is removed from experimental
             if backend == BackendType.OPENVINO:
                 nncf_logger.warning('You are using experimental OpenVINO backend for the Post-training quantization.')
+            if backend == BackendType.TORCH:
+                nncf_logger.warning('You are using experimental Torch backend for the Post-training quantization.')
```

Comment:
I suggest removing it because when we switch `quantize()` to Torch implementation of the post-training quantization algorithm, it will not be an experimental implementation.

Reply:
Agree

Reply:
Done